### PR TITLE
Fix error with 'write() argument must be str, not bytes'

### PR DESCRIPTION
**Pull request by msuksong**
fix the error on python3. works on both python2 and python3

### CODE REVIEW

**Comment by ktbyers** [comment]
@msuksong Thanks. I also need to turn on the unit testing so that Python3 and Ansible gets tested as part of the CI tests.

**Comment by dbarrosop** [comment]
@ktbyers should we merge or do you want to enable py3 tests first?

**Comment by ktbyers** [comment]
@dbarrosop Yes, I don't think this is the right fix for this problem. I will try to come up with a more systematic fix next week.

**Comment by cs-1** [comment]
@ktbyers I'd like to help with fixing this so I tried to understand the root cause of the problem. I'm by no means an expert for napalm or Python but if I see this correctly, the problem is that napalm-ansible takes what it gets from the driver's compare_config(). And if that is a byte stream instead of text, the above problem occurs. My naive approach would be to apply the fix that @msuksong suggested in napalm_install_config.py right after `diff = device.compare_config().encode('utf-8')` to make sure that the diff is str. I'll check whether or not this works and let you know. If I'm completely on the wrong track, please stop me. ;)

**Comment by cs-1** [comment]
The fix in #135 fixes the problem for the platform that I could test it on (napalm_aos using Python 3). However, I can't test it on other platforms so it'd be great if a pro could look at the fix and try it. Thanks.

File: napalm_ansible/modules/napalm_install_config.py [hunk 1]
```diff
@@ -177,7 +177,10 @@
 def save_to_file(content, filename):
     f = open(filename, 'w')
     try:
-        f.write(content)
+        if type(content) is bytes:
```
Comment (dbarrosop) [inline napalm_ansible/modules/napalm_install_config.py:180]: I think `if isinstance(content, bytes)` would be better than comparing types.

Comment (msuksong) [reply napalm_ansible/modules/napalm_install_config.py:180]: agreed!

**Comment by ktbyers** [comment]
Superseded by this PR:

https://github.com/napalm-automation/napalm-ansible/pull/136

**Pull request closed without merging**